native_client/java/app/src/main/java/org/mozilla/deepspeech/DeepSpeechActivity.java

```diff
@@ -49,7 +49,8 @@ private int readLEInt(RandomAccessFile f) throws IOException {
     private void newModel(String tfliteModel) {
         this._tfliteStatus.setText("Creating model");
         if (this._m == null) {
-            this._m = new DeepSpeechModel(tfliteModel, BEAM_WIDTH);
+            this._m = new DeepSpeechModel(tfliteModel);
+            this._m.setBeamWidth(BEAM_WIDTH);
```

Comment:
Given the example, we might just want to get rid of that `setBeamWidth`, we can live with the model's default value.

Reply:
It'll likely change the transcriptions and require adjusting all the constants in the test though.

Reply:
> It'll likely change the transcriptions and require adjusting all the constants in the test though.

It should be a one-time adjustement from moving from 50 to 500, right ? Afterwards, we will be in sync, no?

Reply:
Also, have you tested 500 on Android? Is transcription still fast enough for it to be a sensible default there?

Reply:
Yeah, as much as I remember, it had no big impact. But I was more doing the comment regarding the simplification of the code, if you feel unsafe here, let's keep it, I don't mind :)